scripts/metavar.py

```diff
@@ -1077,23 +1079,30 @@ def write_def(self, outfile, indent, wdict, allocatable=False,
                 raise CCPPError(errmsg.format(name))
             # end if
         # end if
+        optional = self.get_prop_value('optional')
         if protected and dummy:
             intent_str = 'intent(in)   '
         elif allocatable:
             if dimstr or polymorphic:
-                intent_str = 'allocatable  '
+                intent_str = 'allocatable         '
+                if target:
+                    intent_str = 'allocatable,'
```

Comment:
Is this line necessary?

Reply:
It's just to make the auto generated code prettier. 
You get:
allocatable             ::
allocatable, target ::
instead of 
allocatable  ::
allocatable, target ::

Reply:
Sure, I would just use the same sort of logic that is above and below this area. This could be as easy making line 1087 an `else` to the `if` statement on line 1088 and combining 1089 and 1090.

Reply:
I'll defer this to a follow-up PR